JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -2877,6 +2906,18 @@
 }
 
 
+.mmCustomChecks <- list(
+ collinCheck = function(dataset){
+   cor_mat       <- cor(apply(dataset,2,as.numeric))
+   diag(cor_mat) <- 0
+   cor_mat[lower.tri(cor_mat)] <- 0
+   if(any(1 - abs(cor_mat) < 1e-5)){
+     var_ind   <- which(abs(cor_mat) == 1, arr.ind = TRUE)
```

Comment:
```suggestion
   nearOne <- 1 - abs(cor_mat) < sqrt(.Machine$double.eps)
   if(any(nearOne)){
     var_ind   <- which(nearOne, arr.ind = TRUE)
```
The issue is that you check with a tolerance in the if statement but without a tolerance in `which`. 

Using `sqrt(.Machine$double.eps)` (~1.490116e-08) should make this less machine specific, feel free to use a different root though (e.g., `.Machine$double.eps^(1/3)`).